src/components/About/about.tsx

```diff
@@ -0,0 +1,45 @@
+import './about.css';
+
+export const About = () => {
+  return (
+    <div className="about-wrapper">
+      <h2 className="about-title">About</h2>
+      <div className="about-text">
+        <div>
+          About Welcome to our non-profit project, "Exploring the Solar System"! We are Dmytro Honcharov and Oleksii
+          Moroz, two passionate software developers with a shared fascination for the vastness and beauty of our cosmic
+          neighborhood.
+        </div>
+        <p>
+          Our goal is to provide an interactive and educational experience that enables people to grasp the true scale
+          of the Solar System. We believe that understanding the distances between celestial bodies can enhance our
+          appreciation of the universe and our place within it. By leveraging the power of modern web technologies, we
+          aim to make this knowledge easily accessible to everyone.
+        </p>
```

Comment:
why do we have "p" and "div" and what is the diff?

Reply:
done